bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/obscomp/SPSiteQuality.java

```diff
@@ -366,7 +365,8 @@ public double getAdjustment(final BandsList bl) {
         PERCENT_20("20%/Best",  20,  0.5),
         PERCENT_70("70%/Good",  70,  0.0),
         PERCENT_85("85%/Poor",  85, -0.5),
-        ANY(       "Any",      100, -1.0);
+        ANY(       "Any",      100, -1.0),
+        EXACT(     "Exact",      0,  0.0);
```

Comment:
I wonder if this will have an effect on serialization. Would importing old programs still work?

Reply:
This entry will show on the conditions selector on the UI

Reply:
Importing from XML should still work, and I believe just adding new enum members is even binary compatible.  I'm wondering about all the usages though.  For example here:

https://github.com/gemini-hlsw/ocs/blob/develop/bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/api/AgsAnalysis.scala#L196

`EXACT` will be considered `worseOrEqual` to `ANY`.

Reply:
Ah, right.  We should probably not allow selecting "EXACT" in the OT for that reason (AGS) as well.

Reply:
Yes, ProbesLimit has one of those usages too

Reply:
With the enum we get things like:
![image(2)](https://user-images.githubusercontent.com/3615303/130799748-77e5e18f-d6d8-4080-a64e-2a4f50c46f57.png)